server/locale/DE/GPL/styles/flex/base.css

```
@@ -0,0 +1,71 @@
+.message__logo-container {
+    display: flex;
+    align-items: center;
+}
+
+.message__headline {
+    font-family: 'PayPal-Sans-Big';
+    font-size: 8.4vw;
+    line-height: 1.6em;
+    padding: 0px;
+}
+
+.message__disclaimer {
+    position: static;
+    font-family: 'PayPal-Sans-Big';
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0em;
+    text-align: left;
+}
+
+.message__disclaimer > span {
+    font-weight: 400;
+    text-decoration: none;
+    white-space: normal;
+}
+
+.message__disclaimer > span > span {
+    white-space: nowrap;
+}
+
+.message__disclaimer > span > span:only-child,
+.message__disclaimer > span > span.em {
+    text-decoration: underline;
+}
+
+.message__logo:nth-of-type(1) {
+    width: 27px;
+    display: inline-block;
+    /*margin-right: 10px;*/
```

We can comment this back in.
```suggestion
    margin-right: 10px;
```